apps/token-manager/app/src/screens/Holders.js

```diff
@@ -1,166 +1,163 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
 import PropTypes from 'prop-types'
+import { useNetwork } from '@aragon/api-react'
 import styled from 'styled-components'
 import {
+  ContextMenu,
+  ContextMenuItem,
+  DataView,
+  IconAdd,
+  IconLabel,
+  IconRemove,
+  Split,
   TabBar,
   Table,
   TableHeader,
   TableRow,
   Viewport,
   breakpoint,
+  useTheme,
 } from '@aragon/ui'
-import HolderRow from '../components/HolderRow'
-import SideBar from '../components/SideBar'
+import { formatBalance } from '../utils'
+import InfoBoxes from '../components/InfoBoxes'
+import LocalIdentityBadge from '../components/LocalIdentityBadge/LocalIdentityBadge'
+import { useIdentity } from '../components/IdentityManager/IdentityManager'
 
-const TABS = ['Holders', 'Token Info']
+function Holders({
+  groupMode,
+  holders,
+  maxAccountTokens,
+  onAssignTokens,
+  onRemoveTokens,
+  tokenAddress,
+  tokenDecimalsBase,
+  tokenName,
+  tokenSupply,
+  tokenSymbol,
+  tokenTransfersEnabled,
+  userAccount,
+}) {
+  const network = useNetwork()
+  return (
+    <Split
+      primary={
+        <DataView
+          mode="table"
+          fields={groupMode ? ['Owner'] : ['Holder', 'Balance']}
+          entries={holders.map(({ address, balance }) =>
+            groupMode ? [address] : [address, balance]
+          )}
+          renderEntry={([address, balance]) => {
```

Comment:
One thing that I'm not sure whether it's possible to style, is that the [balance in the designs](https://www.figma.com/file/M9AbLkUkeRCFKQt6qdFUsCks/Tokens?node-id=1%3A2) is right-aligned while currently the app has it center-aligned

Reply:
It was a spacing bug in DataView, thanks for having noticed it :hugs: https://github.com/aragon/aragon-ui/pull/391/commits/1e82aec0ae52b29277e8ff2cbe0f4739b20cf79f